### ISSUE TITLE
E3693 rule stating 'AutoMinorVersionUpgrade' isn't valid for Aurora DB clusters

### ISSUE THREAD

**Issue by rwmason** [aws-cloudformation/cfn-lint#3521]
### CloudFormation Lint Version

1.8.0

### What operating system are you using?

Mac OS ventura 13.6.7

### Describe the bug

When running cfn-lint on a CFN template that contains a `AWS::RDS::DBCluster` resource configured to use the `aurora-mysql` engine type, if the `AutoMinorVersionUpgrade` property is specified for the cluster then CFN returns the below warning:

`E3693 Additional properties are not allowed 'AutoMinorVersionUpgrade' when creating Aurora cluster`

The [schema](https://github.com/aws-cloudformation/cfn-lint/blob/main/src/cfnlint/data/schemas/extensions/aws_rds_dbcluster/aurora.json) for this does appear to show this as an invalid property when the engine is set to aurora-mysql or aurora-postgresql, so the rule itself looks to be functioning as expected, however this schema doesn't align with guidance/docs from RDS as described below in the expected behavior.

Additionally, despite the warning that this value is not allowed, there are no issues when creating or updating a stack with `AutoMinorVersionUpgrade` for an Aurora DB cluster, and changing the value for this property is reflected properly for the cluster.

### Expected behavior

The expected behavior is that this would not give a warning as per the RDS documentation this is a valid property for an Aurora DB cluster.

The Aurora RDS docs [here](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html#Aurora.CreateInstance.Settings) list the available settings for an Aurora DB cluster, and the first one listed is for the auto minor version upgrade setting.

Furthermore, elsewhere in the docs for Aurora MySQL found [here](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_UpgradeDBInstance.Maintenance.html#Aurora.Maintenance.AMVU), it also specifically states that this should be set on the DB cluster itself: `We strongly recommend that for new and existing DB clusters, you apply this setting to the DB cluster and not to the DB instances in the cluster individually. If any DB instance in your cluster has this setting turned off, the DB cluster isn't automatically upgraded.`

### Reproduction template

```
Parameters:
  pDBCredsSecret:
    Type: String
    Default: rds/test/credentials
  pVpc:
    Type: AWS::EC2::VPC::Id
  pSubnets:
    Type: List<AWS::EC2::Subnet::Id>
  pAvailabilityZones:
    Type: List<AWS::EC2::AvailabilityZone::Name>

Resources:
  RDSCluster:
    Type: 'AWS::RDS::DBCluster'
    Properties:
      AutoMinorVersionUpgrade: false
      AvailabilityZones: !Ref pAvailabilityZones
      MasterUsername: !Sub '{{resolve:secretsmanager:${pDBCredsSecret}:SecretString:username}}'
      DBSubnetGroupName: !Ref DatabaseSubnetGroup
      MasterUserPassword: !Sub '{{resolve:secretsmanager:${pDBCredsSecret}:SecretString:password}}'
      Engine: aurora-mysql
      EngineVersion: 5.7.mysql_aurora.2.11.5
      VpcSecurityGroupIds:
        - Ref: AuroraSG

  AuroraSG:
    Type: 'AWS::EC2::SecurityGroup'
    Properties:
      GroupDescription: 'RDS security group'
      VpcId: !Ref pVpc
      SecurityGroupIngress: []

  DatabaseSubnetGroup:
    Type: 'AWS::RDS::DBSubnetGroup'
    Properties:
      DBSubnetGroupDescription: 'Subnet group for the Aurora Cluster.'
      SubnetIds: !Ref pSubnets
```

**Comment by kddejong** [comment]
Also do you know why it also listed [here](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html#Aurora.CreateInstance.Settings)
